probably/pgcl/parser.py

```diff
@@ -75,13 +98,15 @@
     %import common.WS
 """
 
+# The order of the operators corresponds to their precedence (earlier operators have lower precedence)
```

Comment:
Here you can refer to the documentation of the `util.lark_expr_parser` module. That doesn't actually explain anything right now though, that should be changed 😅